src/prune_builds

```diff
@@ -99,20 +136,22 @@ if skip_pruning:
 
 # now delete other build dirs not in the manifest
 
-for build_dir, _ in builds_to_delete:
+for build_dir, _, _ in builds_to_delete:
     shutil.rmtree(os.path.join(builds_dir, build_dir))
```

Comment:
One more minor nit, I think this would be clearer as:

```
for build in builds_to_delete:
  shutil.rmtree(os.path.join(builds_dir, build.id))
```

But otherwise LGTM, clear to me to merge with (or without) that tweak.